README.md

```diff
@@ -1,6 +1,9 @@
 # PyoMyo
 Python module for the Thalmic Labs Myo armband. 
 
+##here is a new example.
```

Comment:
The code is awesome

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Readable, Efficient, Reusable